templates/README.md.mustache

```diff
@@ -17,22 +17,38 @@
 [gitter-shield]: https://img.shields.io/badge/chat-on%20gitter-%23c1272d.svg
 [gitter-link]: https://gitter.im/coq-community/Lobby
 
-{{ project_description }}
+{{ description }}
 
 ## Meta
 
-- Initial author(s): {{ initial_authors }}
-- Coq-community maintainer(s): {{ maintainers_names_and_nicknames }}
-- License: [{{ license_name }}]({{ license_file }})
-- Compatible Coq versions: {{ coq_versions }}
-- {{# dependencies }}Additional dependencies: {{ dependencies }}{{/ dependencies }}
+- Initial author(s):
+{{# authors }}
+  - {{ name }}
+{{/ authors }}
+- Coq-community maintainer(s):
+{{# maintainers }}
+  - [{{ name }}](https://github.com/{{ nickname }}) (**@{{ nickname }}**)
+{{/ maintainers }}
+{{# license }}
+- License: [{{ fullname }}]({{ file }}{{^ file }}LICENSE{{/ file }})
+{{/ license }}
+{{# supported_coq_versions }}
+- Compatible Coq versions: {{ text }}
+{{/ supported_coq_versions }}
+{{# plugin }}
+- Compatible OCaml versions: all versions supported by Coq
+{{/ plugin }}
+- Additional dependencies:{{# dependencies }}
+  - {{ description }}
+{{/ dependencies }}{{^ dependencies }} none{{/ dependencies }}
 
 ## Building instructions
 
 ``` shell
-git clone https://github.com/coq-community/{{ no_space_project_name }}
-cd {{ no_space_project_name }}
+git clone https://github.com/coq-community/{{ shortname }}
+cd {{ shortname }}
 make   # or make -j <number-of-cores-on-your-machine>
+make install
 ```
```

Comment:
It's good to have manual build instructions, but don't we want to push for installation via OPAM whenever that is available? Or should that always be in the "optional section"? In many cases, I just look at READMEs to figure out the OPAM package name (which arguably warrants its own variable name in the template).

Reply:
Yes, I like the part you added to the aac-tactics README. I just forgot to report it here.